src/components/views/settings/index.tsx

```diff
@@ -140,6 +164,37 @@ export const Settings: React.FC<SettingsProps> = ({navigation}) => {
 
   const version = getReadableVersion();
 
+  const handleButtonPressed = (screen: string) =>
+    screen === 'feedback'
+      ? Linking.openURL(
+          `mailto:covidalertny@health.ny.gov?subject=App Feedback for COVID Alert NY: Version ${version}&body=<br><br><br><br>
```

Comment:
We're getting different new line handling between email apps, some respect `<br>` and ignore whitespace line breaks or `\n`, some include `<br>` as plain text and respect whitespace line breaks or `\n`.

My suggestion is, that we can deal with whatever formatting we get but we should make sure the user always has space to type by putting `\n\n\n\n` before the `<br><br><br><br>`, so that on some email clients there is whitespace then a block of debugging info, and on others there is whitespace then a harmlessly out-of-the-way line of `<br><br><br><br>` then debugging info.

<img width=250 src="https://user-images.githubusercontent.com/29628323/92921461-fcc8a380-f42b-11ea-8a74-cdfac5d77267.jpg" />
<img width=250 src="https://user-images.githubusercontent.com/29628323/92921463-fd613a00-f42b-11ea-8b9f-8bd89f46316f.jpg" />
<img width=250 src="https://user-images.githubusercontent.com/29628323/92921465-fd613a00-f42b-11ea-92e5-d90edd4aa3d5.jpg" />